.travis/setup-windows.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh -ex
+
+# Git bash doesn't have rev, so here it is
+rev()
+{
+    echo "$1" | awk '{ for(i = length($0); i != 0; --i) { a = a substr($0, i, 1); } } END { print a }'
+}
+
+# Globals
+QT_BASE_URL='http://qt.mirror.constant.com/online/qtsdkrepository/windows_x86/desktop/qt5_5141/qt.qt5.5141.win64_msvc2017_64/5.14.1-0-202001240957qtbase-Windows-Windows_10-MSVC2017-Windows-Windows_10-X86_64.7z'
```

Comment:
Would be nice if we had a QTVER and QTDATE env variable so we could do something like
```bash
QTVER=5.14.1
DATE=202001240957
QT_WIN_URL="http://qt.mirror.constant.com/online/qtsdkrepository/windows_x86/desktop/qt5_${QTVER//./}/qt.qt5.${QTVER//./}.win64_msvc2017_64/${QTVER}-0-${DATE}qtbase-Windows-Windows_10-MSVC2017-Windows-Windows_10-X86_64.7z"
```
And that way updating is a breeze.